src/performance.ts

```diff
@@ -12,17 +12,21 @@ type PerfResult = {
 }
 
 type PerfHighlights = {
-  configLoadTime: number;
-  runTime: number;
-  initTime: number;
-  commandLoadTime: number;
-  commandRunTime: number;
+  'oclif.configLoadMs': number;
+  'oclif.runMs': number;
+  'oclif.initMs': number;
+  'oclif.commandLoadMs': number;
+  'oclif.initHookMs': number;
+  'oclif.prerunHookMs': number;
+  'oclif.postrunHookMs': number;
+  'oclif.commandRunMs': number;
+  'oclif.corePluginsLoadMs': number;
+  'oclif.userPluginsLoadMs': number;
+  'oclif.linkedPluginsLoadMs': number;
   pluginLoadTimes: Record<string, {duration: number, details: Details}>;
```

Comment:
telemetry.record will skip object types and not record them to the telemetry file